lib/clients/reports.ex

```diff
@@ -68,4 +68,84 @@ defmodule GlassFactoryApi.Clients.Reports do
       {:error, message} -> raise message
     end
   end
+
+  @doc """
+  List rates and costs reports from the given client id
+
+  ## Examples
+
+      iex> GlassFactoryApi.Clients.Reports.list_rates_and_costs_reports(1234, [], config)
+      [:ok, %RatesAndCostsReport{
```

Comment:
Shouldn't this be a `{:ok, [...]}` tuple?

Reply:
Oops, my fault 😅, it is just an error on the example, I'll fix it, Thanks ❤️